syntax = "proto3";

service MoeShard {
  rpc Execute (Inputs) returns (Outputs) {}
  rpc ToNextBlock (Empty) returns (Empty) {}
}

message Inputs {
  bytes activated_experts = 2;
  bytes data = 3;
}

message Outputs {
  bytes data = 1;
}

message Empty {}
